modules/heat_conduction/src/kernels/ADHeatConduction.C

```diff
@@ -17,18 +17,34 @@ defineADValidParams(
     params.addParam<MaterialPropertyName>("thermal_conductivity",
                                           "thermal_conductivity",
                                           "the name of the thermal conductivity material property");
+    params.addParam<RealVectorValue>("axis_scaling_vector",
```

Comment:
supplying a default value of `(1,1,1)` does not work here?

Reply:
I tried different ways, but couldn’t get it to work since it’s a weird ad implementation.

Reply:
This is the syntax:
```
params.addParam<RealVectorValue>("axis_scaling_vector", RealVectorValue(1, 1, 1), "Doco...");
```

Reply:
this works...Damnit! I thought I tried, that...